### ISSUE TITLE
Column resize area should be larger

### ISSUE THREAD

**Issue by Sytten** [caido/caido#272]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

Currently the area is quite small and unforgiving, took me a few tries to resize my column (on native windows at least)
